vetlexicon/parser.js

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+
+'use strict';
+const Parser = require('../.lib/parser.js');
+
+/**
+ * Recognizes the accesses to the platform Vetlexicon
+ * @param  {Object} parsedUrl an object representing the URL to analyze
+ *                            main attributes: pathname, query, hostname
+ * @param  {Object} ec        an object representing the EC whose URL is being analyzed
+ * @return {Object} the result
+ */
+module.exports = new Parser(function analyseEC(parsedUrl, ec) {
+  let result = {};
+  let path   = parsedUrl.pathname;
+  // uncomment this line if you need parameters
+  // let param = parsedUrl.query || {};
+
+  // use console.error for debuging
+  // console.error(parsedUrl);
+
+  let match;
+
+  if ((match = /^\/treat\/([a-z]+)\/([a-z-]+)\/([a-z-]+)$/i.exec(path)) !== null) {
+    // https://www.vetlexicon.com/treat/canis/breeds-pages/anatolian-shepherd-dog
```

Comment:
After browsing the platform, for this parser, 'anatolian-shepherd-dog' is enough for unitid. catch only this and use it in unitid. I will merge that and update that too